UPGRADE-6.3.md

```diff
@@ -0,0 +1,89 @@
+UPGRADE FROM 6.2 TO 6.3
+=======================
+
+# Essentials
+
+## Changed Signatures
+
+See the Changed Interfaces and Method Signatures section if changes in signatures affect the project.
+
+# Changed Interfaces and Method Signatures
+
+This section contains information on interface and method signature changes which affect backwards compatibility (BC).
+Note that ONLY BC breaking changes are listed, according to our backwards compatibility policy.
+
+## ChameleonSystem\CoreBundle\DataAccess\CmsPortalDomainsDataAccess
+
+- New method `getActivePortalCandidate()`.
+- New method `getDomainDataByName()`.
+- New method `getPortalPrefixListForDomain()`.
+
+## ChameleonSystem\CoreBundle\DataAccess\CmsPortalDomainsDataAccessCacheDecorator
+
+- New method `getActivePortalCandidate()`.
+- New method `getDomainDataByName()`.
+- New method `getPortalPrefixListForDomain()`.
+
+## ChameleonSystem\CoreBundle\DataAccess\CmsPortalDomainsDataAccessInterface
+
+- New method `getActivePortalCandidate()`.
+- New method `getDomainDataByName()`.
+- New method `getPortalPrefixListForDomain()`.
+
+## ChameleonSystem\CoreBundle\DataAccess\CmsPortalDomainsDataAccessRequestLevelCacheDecorator
+
+- New method `getActivePortalCandidate()`.
+- New method `getDomainDataByName()`.
+- New method `getPortalPrefixListForDomain()`.
```

Comment:
The new methods in the implementations do not really break BC, do they? If we just want to list BC breaking changes, only the interfaces are relevant.

Reply:
I just saw that this change belongs to PR #28 - could you have a look at that PR before merging this one (as requested in the PR description)? Thanks!

Reply:
OK, I've moved my comment there and will resolve this.